package/MDAnalysis/core/AtomGroup.py

```diff
@@ -69,13 +69,13 @@
 The same is mostly true for :class:`Residue` instances although they are
 derived from :class:`Atom` instances: all :class:`Atom` objects with the same
 :attr:`Atom.resid` are bundled into a single :class:`Residue` with
-:class:`Residue.id` = *resid*. This means that just changing, say, the residue
+:class:`Residue.resid` = *resid*. This means that just changing, say, the residue
```

Comment:
Thanks for doing these. I completely forgot to check all the docs for old usage patterns.